toolchain/sem_ir/formatter.cpp

```diff
@@ -1027,13 +1028,13 @@ class FormatterImpl {
             symbolic_constant.index
                 .region()))[symbolic_constant.index.index()]);
         out_ << " (";
-        FormatName(sem_ir_.constant_values().GetInstId(id));
+        FormatName(sem_ir_.constant_values().GetInstIdIfValid(id));
         out_ << ")";
         return;
       }
     }
 
-    FormatName(sem_ir_.constant_values().GetInstId(id));
+    FormatName(sem_ir_.constant_values().GetInstIdIfValid(id));
```

Comment:
These changes look redundant: we already checked for an invalid `id` at the start of this function.

Reply:
Undone here, I admit I didn't look closely on the crash.